.github/workflows/build.yaml

```diff
@@ -38,20 +38,24 @@ jobs:
         uses: actions-rust-lang/rustfmt@v1
 
       - name: Create k8s Kind Cluster
+        if: ${{ false }}
         uses: helm/kind-action@v1
         with:
           cluster_name: 'kind'
 
       - name: Check kind cluster
+        if: ${{ false }}
```

Comment:
not sure why?

Reply:
Disabled temporary, until integration environment is ready.